packages/api/src/axelar-query/types.ts

```diff
@@ -20,3 +28,83 @@ export type EstimateGasFeeResponse = {
   apiResponse: string;
   isExpressSupported: boolean;
 };
+
+export type Environment = "devnet" | "testnet" | "mainnet";
+
+interface TokenPrice {
+  usd: number;
+}
+
+interface GasPriceInUnits {
+  value: string;
+  decimals: number;
+}
+
+interface BaseToken {
+  name: string;
+  symbol: string;
+  decimals: number;
+  token_price: TokenPrice;
+}
+
+interface NativeToken extends BaseToken {
+  contract_address: string;
+  gas_price: string;
+  gas_price_gwei: string;
+  gas_price_in_units: GasPriceInUnits;
+  l1_gas_oracle_address: string;
+  l1_gas_price_in_units: GasPriceInUnits;
+}
+
+export interface BaseFeeResponse {
+  success: boolean;
+  error: unknown;
+  baseFee?: string;
+  expressFee?: string;
+  sourceToken?: NativeToken;
+  executeGasMultiplier?: number;
+  destToken?: {
+    gas_price: string;
+    decimals: number;
+    token_price: TokenPrice;
+    name: string;
+    symbol: string;
+    l1_gas_oracle_address?: string;
+    l1_gas_price_in_units: GasPriceInUnits;
+  };
+  l2_type?: "op" | "arb" | "mantle" | undefined;
+  ethereumToken?: BaseToken;
+  apiResponse?: AxelarGMPResponse;
+  expressSupported?: boolean;
+}
+
+export interface GetNativeGasBaseFeeOptions {
+  sourceTokenSymbol?: string;
+  symbol?: string;
+  destinationContractAddress?: string;
+  sourceContractAddress?: string;
+  amount?: number;
+  amountInUnits?: bigint | string;
+}
+
+export interface AxelarGMPResult {
+  source_base_fee_string: string;
+  source_token: NativeToken;
+  ethereum_token: BaseToken;
+  destination_native_token: NativeToken;
+  express_fee_string: string;
+  express_supported: boolean;
+  l2_type: "op" | "arb" | "mantle" | undefined;
+  execute_gas_multiplier: number;
+}
+
+export interface AxelarGMPResponse {
+  result: AxelarGMPResult;
+}
+export interface AxelarQueryService {
+  readonly evm: EvmQS;
+  readonly axelarnet: AxelarnetQS;
+  readonly nexus: NexusQS;
+  readonly tss: TSSQS;
+}
+export type AxelarQueryClientType = QueryClient & AxelarQueryService;
```

Comment:
can you please remove interfaces/types that we haven't used in this PR?

Reply:
all the types left here are being used

Reply:
thanks for the check. there're some fields duplicated with some types [here](https://github.com/axelarnetwork/axelarjs/blob/main/packages/api/src/gmp/types.ts). can you check if there's anyway we can combine or reuse them?